hoomd/pytest/test_box_resize.py

```diff
@@ -233,3 +234,19 @@ def test_pickling(simulation_factory, two_particle_snapshot_factory,
                   box_resize):
     sim = simulation_factory(two_particle_snapshot_factory())
     operation_pickling_check(box_resize, sim)
+
+
+def test_mutability_error(simulation_factory, two_particle_snapshot_factory):
+    filt = hoomd.filter.All()
+    sim = simulation_factory(two_particle_snapshot_factory())
+    trig = hoomd.trigger.Periodic(1)
+    box1 = hoomd.Box.cube(L=10)
+    box2 = hoomd.Box.cube(L=12)
+    var = hoomd.variant.Ramp(10, 12, 0, 100)
+    box_op = hoomd.update.BoxResize(box1, box2, var, trig)
```

Comment:
```suggestion
    box_op = hoomd.update.BoxResize(box1=box1, box2=box2, variant=var, trigger=trig)
```
The `master` branch changed the order of these arguments, causing the test to fail. Use keyword arguments so they are passed without ambiguity.